staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -138,11 +148,20 @@ func (s *store) Get(ctx context.Context, key string, resourceVersion string, out
 		return storage.NewInternalError(err.Error())
 	}
 
-	return decode(s.codec, s.versioner, data, out, kv.ModRevision)
+	return decode(s.codec, s.versioner, data, out, kv.ModRevision, clusterName)
 }
 
 // Create implements storage.Interface.Create.
 func (s *store) Create(ctx context.Context, key string, obj, out runtime.Object, ttl uint64) error {
+	clusterName := ""
```

Comment:
Style thing, but you should probably do it now.  golang and kube style would always be to init variables that you're using the default value like:

```
var clusterName string
```

or for an int `foo := 0` it hsould be `var foo int`.